src/VisualStudio/Core/Def/ProjectSystem/VisualStudioWorkspaceImpl.AddMetadataReferenceUndoUnit.cs

```diff
@@ -36,9 +37,10 @@ public override void Do(IOleUndoManager pUndoManager)
 
                     if (reference == null)
                     {
+                        var documentationProvider = Workspace.Services.GetRequiredService<IDocumentationProviderService>();
                         try
                         {
-                            reference = MetadataReference.CreateFromFile(_filePath);
+                            reference = MetadataReference.CreateFromFile(_filePath, documentation: documentationProvider.GetDocumentationProvider(_filePath));
```

Comment:
This doesn't actually matter -- the underlying call to workspace.TryApplyChanges doesn't use this instance but rather calls to the project system and we'll create a new reference.

Reply:
I can change this to DocumentationProvider.Default with a comment so it's still explicit but no overhead.